tensorflow/java/src/main/java/org/tensorflow/op/PrimitiveOp.java

```diff
@@ -24,6 +24,13 @@
  * PrimitiveOp}. Custom operations working with only one primitive may also derive from this class.
  */
 public abstract class PrimitiveOp implements Op {
+  
+  /**
+   * Returns the underlying {@link Operation}
```

Comment:
Are the concerns that led to the "deliberately not exposed" comment previously no longer valid?
Or if there is a conflict in a future op, would the code generator have to work around that?

Reply:
>Are the concerns that led to the "deliberately not exposed" comment previously no longer valid?

Exactly, now that we are using API defs to generate the ops, we can rename the attributes in case of a conflict.